sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -64,7 +64,15 @@ import {
   ReleaseKeyOptions,
   ReleaseKeyResult,
   KeyReleasePolicy,
-  KeyExportEncryptionAlgorithm
+  KeyExportEncryptionAlgorithm,
+  RotateKeyOptions,
+  KeyRotationPolicy,
+  KeyRotationPolicyParameters,
+  SetRotationPolicyOptions,
+  GetRotationPolicyOptions,
+  KeyRotationLifetimeActions,
+  KeyRotationPolicyAttributes,
+  KeyRotationLifetimeActionsTrigger
```

Comment:
Nit: IIRC, TypeScript supports trailing commas. I recommend here to make the diff easier next time. Helps avoid merge conflicts better, too.

Reply:
I agree about trailing commas, it makes everything easier - diffs, sorting, shuffling things around. But fighting https://prettier.io/ on this is a losing battle. 

I _can_ change the config and will add an issue for it to default to trailing commas (and prettier 2.0 changed the default such that it would have generated trailing commas here) but I would want consistency with our other packages if possible. I can make an issue to set this across our codebase.